CMake/Dependencies.cmake

```diff
@@ -151,3 +151,35 @@ endif()
 if(NOT NONET AND NOT DISABLE_ZERO_TIER)
   add_subdirectory(3rdParty/libzt)
 endif()
+
+if(DISCORD_INTEGRATION)
+  message("Retrieving Discord SDK")

```

Comment:
Please move this to `3rdParty/discord` and use `FetchContent_MakeAvailableExcludeFromAll`, similar to how we handle other fetched dependencies. No need for any `message` calls btw. `MakeAvailable` will fail with a fatal error if the download fails. Download logging can be turned on globally by the user with `-DFETCHCONTENT_QUIET=OFF`.

Reply:
Having an issue with moving it to `3rdParty/discord`. It's failing to discover `discord_game_sdk.dll.lib` and instead looking for `discord_game_sdk.lib`, but doesn't do that when all that is in `Dependencies.cmake`. Will investigate tomorrow.